ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcLogAppender.java

```diff
@@ -222,6 +222,10 @@ private void increaseNextIndex(AppendEntriesRequestProto request) {
     }
   }
 
+  private void increaseNextIndex(final long installedSnapshotIndex) {
+    getFollower().increaseNextIndex(installedSnapshotIndex + 1);
```

Comment:
FollowerInfo#increaseNextIndex will throw error if current next index is greater than installedSnapshotIndex + 1. I think we should do something like updateCommitIndex here.

Reply:
Done.